aad/spring-cloud-azure-starter-active-directory-b2c/aad-b2c-resource-server/CONVERT_SAMPLE_TO_USE_SPRING_BOOT_3.md

```diff
@@ -0,0 +1,64 @@
+# 1 Convert sample to use Spring Boot 3
+
+## 1.1 Update your infra
+
+- [JDK17](https://www.oracle.com/java/technologies/downloads/).
+
+## 1.2 Update Java code
+
+- Update code for the class `com.azure.spring.sample.aad.b2c.security.WebSecurityConfiguration`.
+
+    Current for Spring Boot 2:
+
+    ```java
+    import com.azure.spring.cloud.autoconfigure.aad.AadJwtBearerTokenAuthenticationConverter;
+    import org.springframework.security.config.annotation.method.configuration.EnableGlobalMethodSecurity;
+    import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+    import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+    import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+    
+    @EnableWebSecurity
```

Comment:
How about moving this into `src/main/java` and commend out some codes out like `@EnableWebSecurity` and `@EnableGlobalMethodSecurity`  to make these codes not take effect.

And in `CONVERT_SAMPLE_TO_USE_SPRING_BOOT_3.md`, let customer remove `//` manually.

Reply:
BTW, will CI pass without these changes in Spring Boot 3?

Reply:
After discussion, won't update this.